### PR TITLE
Added some more detail on rustup.

### PR DESCRIPTION
**Pull request by brandonpollack23**
Rusutp is the reccomended way to install rust and maintained by the core team.

I added some organization and suggested that that be the method of installation.

### CODE REVIEW

File: src/cargo.md [hunk 1]
```diff
@@ -5,6 +5,26 @@ used in the Rust ecosystem to build and run Rust applications. Here we want to
...
+## Installation
+
+### Rustup (Recommended)
+
+You can follow the instructions to install cargo and rust compiler, among other standard ecosystem tools with the [rustup][3] tool, which is maintained by the Rust Foundation.
+
+On Unix-like systems you can curl down the install script and pipe it into your shell like so:
+
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```
Comment (mgeisler) [inline src/cargo.md:17]: I would like us to not add this line of magic... I prefer it if there is one place on the Internet with the recommended command. Put differently: we should absolutely mention and recommend `rustup.rs`, but we should let them explain how to install in details.

**Review by mgeisler — CHANGES_REQUESTED** [review]
Thanks for adding this! It's something which was missing — I only provided the Debian command line since I aimed the course at Googlers with gLinux. So thanks for broadening this to the rest of the world.

File: src/cargo.md (hunk 1 above)
Comment (brandonpollack23) [reply src/cargo.md:17]: You are absolutely right, this is subject to change, but a link will always be right

**Comment by brandonpollack23** [comment]
Should be good to go

**Review by mgeisler — APPROVED** [review]

**Pull request merged**